src/services/parameterStoreService.ts

```diff
@@ -0,0 +1,27 @@
+import { SSMClient, GetParameterCommand } from "@aws-sdk/client-ssm";
+
+interface BannerConfig {
```

Comment:
Do you think we should set this up so that:
- You can configure multiple banners at the same time?
- You can configure each banner to appear on multiple pages?

Reply:
Hm, need to extend this for multiple page/banner support. Thanks